Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -129,57 +133,49 @@ - (instancetype)initWithFrame:(CGRect)frame {
  Microsoft Extension
 */
 - (instancetype)initWithFrame:(CGRect)frame xamlElement:(RTObject*)xamlElement {
-    // TODO: We're passing nil to initWithFrame:xamlElement: because we have to *contain* a _subview for padding/layout.
-    // Note: Pass 'xamlElement' instead, once we move to a *single* backing Xaml element for UIActivityIndicatorView.
-    if (self = [super initWithFrame:frame xamlElement:nil]) {
-        [self _initUIActivityIndicatorView:xamlElement];
+    if (self = [super initWithFrame:frame xamlElement:xamlElement]) {
+        [self _initUIActivityIndicatorView];
     }
 
     return self;
 }
 
 /**
  @Status Interoperable
+ @Note Frame will not be resized smaller than the contained ProgressRing, as this would obscure it.
```

Comment:
iOS allows it to be resized smaller than the indicator, but still displays the indicator and the frame. For example, if you were to set the background color to black, and set the indicator frame to be 5x5. You would see the spinning indicator (normal size is 20x20) with a small 5x5 black square inside of it. In XAML, if the dimensions of the containing frame are smaller than the ProgressRing inside, it will be partially obscured.